### PR TITLE
Add c++ proto support

### PR DESCRIPTION
**Pull request by sjerome**
Add c++ proto support to tensborboard.

### CODE REVIEW

File: tensorboard/defs/protos.bzl [hunk 1]
```diff
@@ -13,8 +13,16 @@
 # limitations under the License.
 
 load("@protobuf//:protobuf.bzl", "py_proto_library")
+load("@protobuf//:protobuf.bzl", "cc_proto_library")
```
Comment (chihuahua) [inline tensorboard/defs/protos.bzl]: nit: Load these 2 in one `load` command.

File: tensorboard/defs/protos.bzl [hunk 2]
```diff
@@ -13,8 +13,16 @@
...
 
 def tb_proto_library(name, srcs = [], visibility = []):
+  cc_proto_library(
```
Comment (chihuahua) [inline tensorboard/defs/protos.bzl]: Apparently, this call needs `default_runtime = "@protobuf//:protobuf",` for TensorBoard to build. ... yeah ... :/ I don't know exactly why.

File: tensorboard/defs/protos.bzl [hunk 3]
```diff
@@ -13,8 +13,16 @@
...
+    name,
+    srcs=srcs,
```
Comment (chihuahua) [inline tensorboard/defs/protos.bzl]: nit: Space on both sides of `=` for `srcs` and `visibility`.

**Comment by wchargin** [comment]
We use `proto_gen` now, so I think you should be able to wrap a
`cc_proto_library` around that? In any case, closing since this is three
years stale.

**Pull request closed without merging**